Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.10
+LABEL maintainer "Wes Lambert, wlambertts@gmail.com"
+LABEL version="CyberChef server 0.0.1"
```

Comment:
I don't think this is necessary. If we decide to publish the image to an image registry, we could align the tag on that image with the tag of the application version, which would be easier to maintain and more obvious.

Reply:
I can certainly remove the version label.

Reply:
Updated in PR.